checkout/checkin.md

```diff
@@ -0,0 +1,281 @@
+
+{:.text-center}
+![Consumer UI][checkin-image]{:width="564" height="293"}
+
+As you can see, the payer's information is pre-filled as provided by the
+initial `POST`. With a `consumerProfileRef` safely tucked into our pocket,
+the Checkin is complete and we can move on to [Payment Menu][payment-menu].
+
+A complete overview of how the process of identifying the payer through Checkin
+is illustrated in the sequence diagram below.
+
+```mermaid
+sequenceDiagram
+    participant Payer
+    participant Merchant
+    participant SwedbankPay as Swedbank Pay
+
+        rect rgba(238, 112, 35, 0.05)
+            note left of Payer: Checkin
+
+    Payer ->>+ Merchant: Start Checkin
+    Merchant ->>+ SwedbankPay: POST /psp/consumers
+    deactivate Merchant
+    SwedbankPay -->>+ Merchant: rel:view-consumer-identification ①
+    deactivate SwedbankPay
+    Merchant -->>- Payer: Show Checkin on Merchant Page
+
+    Payer ->>+ Payer: Initiate Consumer Hosted View (open iframe) ②
+    Payer ->>+ SwedbankPay: Show Consumer UI page in iframe ③
+    deactivate Payer
+    SwedbankPay ->>- Payer: Consumer identification process
+    activate Payer
+    Payer ->>+ SwedbankPay: Consumer identification process
+    deactivate Payer
+    SwedbankPay -->>- Payer: show consumer completed iframe
+    activate Payer
+    Payer ->> Payer: EVENT: onConsumerIdentified (consumerProfileRef) ④
+    deactivate Payer
+    end
+```
+
+If a browser refresh is performed after the payer has checked in, the payment
+menu must be shown even though `onConsumerIdentified` is not invoked.
+
+Additional events during Checkin  can also be implemented
+in the `configuration` object, such as `onConsumerIdentified`, `onShippingDetailsAvailable`and
+`onBillingDetailsAvailable`. Read more about these in the
+[Checkin events][checkin-events] section.
+
+### Note on consumer data
+
+During this stage some consumer data is stored.
+Read more about our [Data Protection Policy][data-protection] for details on which
+information we store and its duration.
+
+{% include iterator.html prev_href="./"
+                         prev_title="Back: Introduction"
+                         next_href="payment-menu"
+                         next_title="Next: Payment Menu" %}
+
+[capture-operation]: /checkout/after-payment#capture
+[checkin-image]: /assets/img/checkout/your-information.png
+[checkin-events]: /checkout/other-features#checkin-events
+[consumer-reference]: /checkout/other-features#payee-reference
+[data-protection]: /resources/data-protection#paymentorder-consumer-data
+[initiate-consumer-session]: /checkout/checkin#checkin-back-end
+[msisdn]: https://en.wikipedia.org/wiki/MSISDN
+[operations]: /checkout/other-features#operations
+[order-items]: #order-items
+[payee-reference]: /checkout/other-features#payee-reference
+[payment-menu-image]: /assets/img/checkout/payment-methods.png
+[payment-menu]: #payment-menu
+[payment-menu]: payment-menu
```

Comment:
Codacy found an issue: [[no-duplicate-definitions] Do not use definitions with the same identifier (273:1)](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5659803)